data/changelog/ocaml/2023-08-??-ocaml-5.1.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Release of OCaml 5.1.0
+description: Release of OCaml 5.1.0
+date: "2023-08-??"
+tags: [ocaml, release]
+changelog: |
+---
+
+We have the pleasure of celebrating the birthday of ??? by announcing the release of OCaml version 5.1.0.
+
+Some of the highlights in OCaml 5.1.0 are:
+
+   - Many runtime performance regression fixes (dynlinking, weak array, weak hash sets, GC with idle domains, GC prefetching).
+   - Restored support for RISC-V and s390x architectures.
+   - Restored Cygwin port.
+   - Reduced installation size (50% reduction)
+   - Compressed compilation artefacts (.cmi, .cmt, .cmti, .cmo, .cma files)
```

Comment:
I would note that all OCaml packages, not just the compiler, benefit from this.

Reply:
I have the impression that it would be a bit too heavy-handed for the brief list of highlights.